alfalfa_worker/lib/models.py

```diff
@@ -0,0 +1,262 @@
+import datetime
+import json
+
+from mongoengine import (
+    CASCADE,
+    DateTimeField,
+    DictField,
+    Document,
+    DynamicField,
+    EmbeddedDocument,
+    EmbeddedDocumentField,
+    FloatField,
+    IntField,
+    ListField,
+    ReferenceField,
+    StringField,
+    UUIDField
+)
+
+from alfalfa_worker.lib.sim_type import SimType
+
+
+class TimestampedDocument(Document):
```

Comment:
base class which we inherit all models. this ensures that we have `created` and `modified` fields on all documents.